CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_shopify v0.12.2
+
+[PR #84](https://github.com/fivetran/dbt_shopify/pull/84) includes the following changes:
+
+## Feature
+- Introduce the variable `shopify__calendar_start_date` to `shopify__calendar` to allow for the start date to be customized. This can be set in your `dbt_project.yml`. If not used, the default will start at `2019-01-01`. See the [README](https://github.com/fivetran/dbt_shopify/blob/main/README.md#Change-the-calendar-start-date) for more details. 
```

Comment:
```suggestion
- Introduced the variable `shopify__calendar_start_date` to `shopify__calendar` to allow for the start date to be customized. This can be set in your `dbt_project.yml`. If not used, the default will start at `2019-01-01`. See the [README](https://github.com/fivetran/dbt_shopify/blob/main/README.md#Change-the-calendar-start-date) for more details. 
```